app/views/articles/show.html.erb

```diff
@@ -31,6 +31,53 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": {
+      "@type": "WebPage",
+      "@id": "https://dev.to/<%= article_url(@article) %>"
+    },
+    "url": "<%= article_url(@article) %>",
+    "image": "<%= article_social_image_url(@article) %>",
+    "publisher": {
+      "@context": "http://schema.org",
+      "@type": "Organization",
+      "name": "<%= community_qualified_name %>"
```

Comment:
missing trailing comma

Reply:
I don't believe that I need a comma here since it is the final key-value pair for `publisher`, but I will double-check to make sure of that. 😊

Reply:
ahahha sorry, I think this was meant for the other file and then I forgot to remove it :D

Reply:
No worries! I appreciate any and all of your feedback. 😄